### DISCUSSION TITLE
cannot install os updates

### DISCUSSION THREAD

**Discussion by deltaforce** [elementary/appcenter#1928] [General]
when I try to install OS updates I am getting this error.
what do I do ?

The following packages have unmet dependencies:
  libpantheon-files-core0: Conflicts: libpantheon-files-widgets0 (< 6.1.1) but 6.1.0+r5055+pkg107~ubuntu6.1 is to be installed

**Comment by lenemter** [comment]
Does this issue still persist?

**Reply by deltaforce** [reply]
I was not able to fix the issue, but I found out how to install all the updates via command line instead.

**Reply by lenemter** [reply]
I guess it was some kind of packaging issue that has already been fixed

**Reply by deltaforce** [reply]
Don;t see how it could have been fixed already, I literally downloaded and installed the ISO the day before leaving this post, so I obviously had the latest version.

**Reply by juuljuul-debug** [reply]
> I was not able to fix the issue, but I found out how to install all the updates via command line instead.

Could you post the work around? plz
